src/pal/src/configure.cmake

```diff
@@ -933,6 +933,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
   set(PAL_PT_READ_D PT_READ_D)
   set(PAL_PT_WRITE_D PT_WRITE_D)
   set(HAS_FTRUNCATE_LENGTH_ISSUE 1)
+  set(HAVE_SCHED_OTHER_ASSIGNABLE 1)
```

Comment:
This has nothing to do with cache and we would not need to clean the cache. It updates the variable with `set(VAR 1)` then `set(VAR 2)`, unless you are using cache in `set` which you aren't in this case.